htdocs/admin/user.php

```diff
@@ -184,6 +184,25 @@
 }
 print '</td></tr>';
 
+// user can edit is IBAN in rh tab if he is authorized to edit himself his informations
+
+print '<tr class="oddeven">';
```

Comment:
Permissions to do or not must not be managed by an option. This make things unclear for the permission administrator.

Because IBAN is an information provided by the user. If the user has the permission  user->self->creer , he can be allowed ot modify IBAN.
"user->self->creer" means you can modify all information related to you so using this can be enough.

Sure, having a warning sending an email to warn the change is a good thing, but this can be another PR...